Software/dczia26_final/dczia26_final.ino

```diff
@@ -1,16 +1,30 @@
-// dczia 2018 proto dos test firmware
-// combines all major hardware aspects (led, oled, keypad, ble)
+// dczia 2019 firmware for DCZia26 Badge
+// combines all major hardware aspects (led, oled, keypad, ble, mic)
 
 // split into functional regions
-#include "dczia26.h"
+#include "Key.h" // This was referred to as dczia.h previously
 #include "dczia26_keypad.h"
 #include "dczia26_led.h"
 #include "dczia26_oled.h"
 #include "dczia26_ble.h"
 #include "dczia26_sd.h"
+#include "arduinoFFT.h" // Standard Arduino FFT library 
```

Comment:
Just confirming, this is the `1..5.0` version of the library? It worked when I built it, but just want to book keep it for the README.

Reply:
arduinoFFT 1.5.0 is indeed correct!